simphony_metadata.yml

```diff
@@ -13,41 +13,41 @@ Resources:
 CUDS_KEYS:
 
   CUDS_ITEM:
-    parent: 
+    parent:
     CUBA.UUID:
       scope: CUBA.SYSTEM
-    CUBA.DATA:  
-      
+    CUBA.DATA:
+
   CUDS_COMPONENT:
     definition: Base data type for the CUDS components
     parent: CUBA.CUDS_ITEM
     CUBA.DESCRIPTION:
```

Comment:
right! the description in cuba.yml is the definition in the new approved schema. Definition is on the level of metadata (this as a class documentation if you like), while CUBA.DESCRIPTION is an attribute of the data entity, that is set by the user (an instance documentation, wherebye both should exist).